autoload/ensime.vim

```diff
@@ -3,10 +3,6 @@ if !has('nvim')
     execute ':redraw!'
 endif
 
-function! ensime#client_keys() abort
-    return s:call_plugin('client_keys', [])
-endfunction
```

Comment:
This is a breaking change to the Vim scripting API—I'm making the bold assumption that no one is actually using this function… what it returns is a list of absolute paths to `.ensime` files for possibly multiple projects open in the Vim session. This could potentially be useful, but if so, the name sucks anyway—it's totally an implementation detail that the config file paths are a "key" used for caching client instances internally.

I'd like to put off commitment to stability of our Vimscript API for later; it is ugly right now and should change dramatically whenever we take the time to give its design due consideration (related: #243, #244, #64).